src/App.js

```diff
@@ -1,54 +1,73 @@
 import { useEffect, useState } from "react";
-import './App.css';
-import logo from './mlh-prep.png'
+import "./App.css";
+import logo from "./mlh-prep.png";
+import MapComponent from "./map";
 
 function App() {
   const [error, setError] = useState(null);
   const [isLoaded, setIsLoaded] = useState(false);
-  const [city, setCity] = useState("New York City")
+  const [city, setCity] = useState("New York City");
   const [results, setResults] = useState(null);
+  const [searchedLocation, setSearchedLocation] = useState(null);
 
   useEffect(() => {
-    fetch("https://api.openweathermap.org/data/2.5/weather?q=" + city + "&units=metric" + "&appid=" + process.env.REACT_APP_APIKEY)
-      .then(res => res.json())
+    fetch(
+      "https://api.openweathermap.org/data/2.5/weather?q=" +
+        city +
+        "&units=metric" +
+        "&appid=" +
+        process.env.REACT_APP_APIKEY
```

Comment:
Code improvement suggestion - Use template literals (a.k.a backticks) for string concatenation.

Reply:
done